hs_swat_modelinstance/templates/includes/application_scripts.html

```diff
@@ -0,0 +1,17 @@
+{% if debug %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone.marionette.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/backbone-relational.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/moment-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/jqcsrf.js'></script>
+{% else %}
+<script type="text/javascript" src='{{ STATIC_URL }}js/underscore-min.js'></script>
+<script type="text/javascript" src='{{ STATIC_URL }}js/json2.js'></script>
```

Comment:
No supported browser lacks JSON support.

`json2.js` should be removed entirely.

All seven copies.